per_sdk/solana/airdrop.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import asyncio
+import json
+import logging
+import os
+
+from solana.rpc.async_api import AsyncClient
+from solders.keypair import Keypair
+
+from per_sdk.solana.helpers import read_kp_from_json
+
+logger = logging.getLogger(__name__)
+
+
+async def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-v", "--verbose", action="count", default=0)
+    parser.add_argument(
+        "--rpc-url",
+        type=str,
+        required=False,
+        default="http://localhost:8899",
+        help="URL of the Solana RPC endpoint to use for submitting transactions",
+    )
+    parser.add_argument(
+        "--airdrop-amount",
+        type=int,
+        required=False,
+        default=10**9,
+        help="Amount of lamports to airdrop to the keypairs",
+    )
+    args = parser.parse_args()
+
+    logger.setLevel(logging.INFO if args.verbose == 0 else logging.DEBUG)
+    log_handler = logging.StreamHandler()
+    formatter = logging.Formatter(
+        "%(asctime)s %(levelname)s:%(name)s:%(module)s %(message)s",
+        datefmt="%Y-%m-%d %H:%M:%S",
+    )
+    log_handler.setFormatter(formatter)
+    logger.addHandler(log_handler)
+
+    client = AsyncClient(args.rpc_url)
```

Comment:
use "confirmed" as the default commitment level of the clients to speedup tx confirmations